quail/processes/wps_climdex_get_available_indices.py

```diff
@@ -0,0 +1,182 @@
+    default), the names of the functions corresponding to the indices
+    """
+
+    def __init__(self):
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{
+                "load_rdata": 10,
+                "save_rdata": 90,
+            },
+        )
+        inputs = [
+            climdex_input,
+            ci_name,
+            output_file,
+            vector_name,
+            LiteralInput(
+                "get_function_names",
+                "Get function names",
+                abstract="Whether to return function names",
+                default=True,
+                min_occurs=0,
+                max_occurs=1,
+                data_type="boolean",
+            ),
+            log_level,
+        ]
+
+        outputs = [
+            rda_output,
+            LiteralOutput(
+                "avail_processes",
+                "Available processes dictionary",
+                abstract="Available climdex indices (keys) and the processes to use to compute them (values)",
+                data_type="string",
+            ),
+        ]
+
+        super(ClimdexGetAvailableIndices, self).__init__(
+            self._handler,
+            identifier="climdex_get_available_indices",
+            title="Climdex Get Available Indices",
+            abstract="Returns the names of all the indices which may be computed or, if get_function_names is TRUE, the names of the functions corresponding to the indices.",
+            metadata=[
+                Metadata("NetCDF processing"),
+                Metadata("Climate Data Operations"),
+                Metadata("PyWPS", "https://pywps.org/"),
+                Metadata("Birdhouse", "http://bird-house.github.io/"),
+                Metadata("PyWPS Demo", "https://pywps-demo.readthedocs.io/en/latest/"),
+            ],
+            inputs=inputs,
+            outputs=outputs,
+            store_supported=True,
+            status_supported=True,
+        )
+
+    processes = {
+        "su": "climdex_days",
+        "id": "climdex_days",
+        "txx": "climdex_mmdmt",
+        "txn": "climdex_mmdmt",
+        "tx10p": "climdex_temp_pctl",
+        "tx90p": "climdex_temp_pctl",
+        "wsdi": "climdex_spells",
+        "fd": "climdex_days",
+        "tr": "climdex_days",
+        "tnx": "climdex_mmdmt",
+        "tnn": "climdex_mmdmt",
+        "tn10p": "climdex_temp_pctl",
+        "tn90p": "climdex_temp_pctl",
+        "csdi": "climdex_spells",
+        "rx1day": "climdex_rxnday",
+        "rx5day": "climdex_rxnday",
+        "sdii": "sdii",
+        "r10mm": "climdex_rmm",
+        "r20mm": "climdex_rmm",
+        "rnnmm": "climdex_rmm",
+        "cdd": "climdex_spells",
+        "cwd": "climdex_spells",
+        "r95ptot": "climdex_ptot",
+        "r99ptot": "climdex_ptot",
+        "prcptot": "climdex_ptot",
+        "gsl": "climdex_gsl",
+        "dtr": "climdex_dtr",
+    }
```

Comment:
Have you considered a way to dynamically generate this list? That way we don't have to worry about manually updating it any time we add/change a process.